tasks/install.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Ensure hostname is resolvable
```

Comment:
It needs to be in both, the hostname needs to be set when you install SSMTP and when you use it, the hostname may be different between the install and configure stages because they could be run on different machines (build box and then the deployed box).